trento/adoc/checks_customization.adoc

```diff
@@ -0,0 +1,53 @@
+[[checks_customization]]
+
+== Checks Customization
+
+=== Overview of checks Customization
+
+Trento makes it possible to adjust expected check values to match target-specific environmental requirements. This can be done directly through the Trento Web console without modifying the original check YAML files or creating separate checks for different environments.
```

Comment:
```suggestion
Trento makes it possible to adjust expected check values to match target-specific requirements. This can be done directly through the Trento Web console without modifying the original check or impacting other targets.
```